src/issuer.rs

```diff
@@ -767,6 +756,35 @@ impl Issuer {
         Ok(rev_reg_delta)
     }
 
+    pub fn _update_revocation_accumulator(
```

Comment:
Is there a specific reason for the `_` prefix syntax here?

Reply:
Oops, yes because it's meant to be private.